### DISCUSSION TITLE
Localhost not reachable on debian server

### DISCUSSION THREAD

**Discussion by dwk601** [coollabsio/coolify#3624] [Q&A]
Hi i have a debian server in my home, and I used 
`
curl -fsSL https://cdn.coollabs.io/coolify/install.sh | bash
`

after that it gives the wrong ip address so I just typed the ip address i'm using with :8000 and got in to the dashboard but it won't connect to the localhost server. Anyway to fix this?

**Comment by marcellobonini** [comment]
If you still look for answer, the issue might be the two servers are on separate networks or with firewall setup that denies connecting to each other.
